ui/src/views/Settings.vue

```diff
@@ -23,6 +23,15 @@
       <cv-column>
         <cv-tile light>
           <cv-form @submit.prevent="configureModule">
+            <cv-text-input
+              :label="$t('settings.fqdn')"
+              v-model="fqdn"
+              :placeholder="$t('settings.fqdn')"
+              :disabled="loading.getConfiguration || loading.configureModule"
+              :invalid-message="error.address"
+              :helperText="$t('settings.fqdn_helper')"
```

Comment:
`settings.fqdn_helper` is missing on the i18n translation file. Add the field both in:
- ui/public/i18n/it/translation.json
- ui/public/i18n/en/translation.json